scripts/upload-google-oauth-secrets.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
```

Comment:
Totally agree. As hector would say, we could solve it with KV Store :D

But, I don't think it should be all combined at the current state. Let's say, I'd like to test DataDog on my Dev environment. Do I need to upload all possible credentials, like Google Console API Keys, to my "private" bucket, if I don't need them?